integration/update_test.go

```diff
@@ -466,6 +467,98 @@ func TestUpdateSet(t *testing.T) {
 	}
 }
 
+func TestUpdateTimestamp(t *testing.T) {
+	t.Parallel()
+
+	// store the current timestamp with $currentDate operator;
+	// read it, check that it is close to the current time;
+	// write a new timestamp value with the same time;
+	// read it back, and check that it is still close to the current time.
```

Comment:
Let's move those comments to individual steps

Reply:
> Let's move those comments to individual steps

done.